src/test/fixtures/profiles/processed-profile.js

```diff
@@ -120,6 +120,35 @@ export function addRawMarkersToThread(
   }
 }
 
+function _replaceUniqueStringFieldValuesWithStringIndexesInMarkerPayload(
+  payload: MixedObject | null,
+  markerSchemas: MarkerSchema[],
+  stringTable: UniqueStringArray
+) {
+  if (payload === null) {
+    return;
+  }
+  const markerType = payload.type;
+  if (markerType === undefined) {
+    return;
+  }
+  const schema = markerSchemas.find((schema) => schema.name === markerType);
```

Comment:
would it be convenient to use `getMarkerSchemaName` from marker-schema.js?

Reply:
Unsure. I've decided not to change this, because it would add more code to transform the schema list into an object map and to pass the marker name from the caller.